python/src/aac/lang/hierarchy.py

```diff
@@ -48,13 +41,13 @@ def get_definition_ancestry(definition: Definition, context: LanguageContext) ->
             ancestor_root_schema = get_definition_schema(ancestor_definition, context)
             found_self_defined_ancestor = ancestor_definition == ancestor_root_schema
             ancestors_list.append(ancestor_definition)
-            ancestor_definition = get_root_definition_by_key(ancestor_definition.get_root_key(), context.definitions)
+            ancestor_definition = get_root_definition_by_key(ancestor_definition.get_root_key(), context)
```

Comment:
Swapped to passing the full context because of the changes in `get_root_definition_by_key()`. No longer needing the list of definitions to check against but the new `get_root_definitions()` method instead. Right?

Reply:
It just seemed a bit jankey to extract this method to  a list of definitions in the context when the vast majority of the code base operates on the context.  I thought it created unnecessary complexity.  As things stand, when you need to understand definitions you should use the language context.